packages/rum-core/src/performance-monitoring/capture-hard-navigation.js

```diff
@@ -102,15 +102,17 @@ function createResourceTimingSpan(name, initiatorType, start, end) {
   if (initiatorType) {
     kind += '.' + initiatorType
   }
+  const spanName = stripQueryStringFromUrl(name)
+  const span = new Span(spanName, kind)
+  /**
+   * Add context information when the span name and entry name are different
+   */
+  if (spanName !== name) {
```

Comment:
@vigneshshanmugam , I think we should consider this a breaking change since we're removing a field that users might have relied on!

Reply:
I am not sure here, I don't have any numbers on how much users search for context in kibana search UI. What we can do is to split this change and do in separate PR and not combine with this performance optimisation.

Reply:
I think generally we should keep in mind how each change affects the users, since we're open source we might never know some of the use-cases!

Yeah, let's split to avoid having multiple changes in the same PR.